packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/InteractionLayer.js

```diff
@@ -53,39 +53,42 @@ function InteractionLayer ({
     return svgEventOffset
   }
 
+  function createMark (event) {
+    const mark = activeTool.createMark({
+      id: cuid(),
+      frame,
+      toolIndex: activeToolIndex
+    })
+
+    mark.initialPosition(convertEvent(event))
+    setActiveMark(mark)
+    setCreating(true)
+    mark.setSubTaskVisibility(false)
+  }
+
   function onPointerDown (event) {
+    const { target, pointerId } = event
+    target.setPointerCapture(pointerId)
     if (disabled || move) {
       return true
     }
-    let activeMark
 
     if (!activeTool.type) {
       return false;
     }
 
     if (creating) {
-      activeMark = activeTool.handlePointerDown && activeTool.handlePointerDown(convertEvent(event))
-      if (activeMark.finished) setCreating(false)
+      activeTool.handlePointerDown && activeTool.handlePointerDown(convertEvent(event), activeMark)
+      if (activeMark.finished) onFinish(event)
       return true
     }
 
-    activeMark = activeTool.createMark({
-      id: cuid(),
-      frame,
-      toolIndex: activeToolIndex
-    })
-
-    activeMark.initialPosition(convertEvent(event))
-    setActiveMark(activeMark)
-    setCreating(true)
-    activeMark.setSubTaskVisibility(false)
+    createMark(event)
     return false
   }
 
   function onPointerMove (event) {
     if (creating) {
-      const { target, pointerId } = event
-      target.setPointerCapture(pointerId)
       activeMark.initialDrag(convertEvent(event))
```

Comment:
Maybe move this into the tool models, so that transcription line tools can ignore dragging?